code/modules/medical/genetics/bioEffects/hemochromia.dm

```diff
@@ -0,0 +1,275 @@
+/datum/bioEffect/hemochromia_unknown
+	name = "Hemochromia Type-U"
+	desc = "A volatile mutation with 12 known stable alternatives. Will quickly break down into one of them if the subject exits the scanner."
+	id = "hemochromia_unknown"
+	probability = 111
+	effectType = EFFECT_TYPE_MUTANTRACE
+	msgGain = "You feel like you're out of breath."
+	msgLose = "You feel more stable."
+	reclaim_fail = 10
+	lockProb = 1
+	lockedGaps = 2
+	lockedDiff = 2
+	lockedChars = list("A","T","C","G")
+	lockedTries = 12
+	stability_loss = 5
+	icon_state  = "hemochromia_unknown"
+
+	OnLife()
+		if(..()) return
+		var/mob/living/carbon/human/H = owner
+		if(ishuman(owner) && prob(12) && !istype(H.loc, /obj/machinery/genetics_scanner))
+			if(holder.GetASubtypeEffect(/datum/bioEffect/hemochromia))
+				holder.RemoveEffect(src.id)
+				holder.RemovePoolEffect(src)
+				return
+			var/typeRange = 0
+			if(rand(13) <= 4)
+				switch(H.bioHolder.bloodType)
+					if("A+")
+						if(prob(50))
+							typeRange = 1
+						else
+							typeRange = 2
+					if("A-") typeRange = 3
+					if("B+") typeRange = 4
+					if("B-") typeRange = 5
+					if("AB+") typeRange = 6
+					if("O-") typeRange = 7
+					if("O+") typeRange = 8
+					if("AB-")
+						if(prob(50))
+							typeRange = 9
+						else
+							typeRange = 10
+					else 
+						if(prob(100/3))
+							typeRange = 12
+						else
+							typeRange = 11
+			else
+				typeRange = rand(1,12)
+			var/datum/bioEffect/NEW = null
+			switch(typeRange)
+				if(2) NEW = new /datum/bioEffect/hemochromia/bronze()
+				if(3) NEW = new /datum/bioEffect/hemochromia/gold()
+				if(4) NEW = new /datum/bioEffect/hemochromia/lime()
+				if(5) NEW = new /datum/bioEffect/hemochromia/olive()
+				if(6) NEW = new /datum/bioEffect/hemochromia/jade()
+				if(7) NEW = new /datum/bioEffect/hemochromia/teal()
+				if(8) NEW = new /datum/bioEffect/hemochromia/cobalt()
+				if(9) NEW = new /datum/bioEffect/hemochromia/indigo()
+				if(10) NEW = new /datum/bioEffect/hemochromia/purple()
+				if(11) NEW = new /datum/bioEffect/hemochromia/violet()
+				if(12) NEW = new /datum/bioEffect/hemochromia/fuschia()
+				else NEW = new /datum/bioEffect/hemochromia/rust()
+
+			H.bioHolder.AddEffectInstance(NEW,1)
+			holder.RemoveEffect(src.id)
+			holder.RemovePoolEffect(src)
+
+ABSTRACT_TYPE(/datum/bioEffect/hemochromia)
+
+/datum/bioEffect/hemochromia
+	name = "Hemochromia Type-A"
+	desc = "An abstract mutation which is so much more unstable than Type-U that it shouldn't even be possible to observe with current technology."
+	id = "hemochromia_abstract"
+	probability = 0
+	effectType = EFFECT_TYPE_MUTANTRACE
+	msgGain = "You feel a bit less soft."
+	msgLose = "You feel lightheaded for a moment."
+	reclaim_fail = 10
+	lockProb = 1
+	lockedGaps = 2
+	lockedDiff = 2
+	lockedChars = list("A","T","C","G")
+	lockedTries = 12
+	stability_loss = 5
+	icon_state  = "hemochromia_unknown"
+	occur_in_genepools = 0
```

Comment:
```suggestion
	occur_in_genepools = 0
	acceptable_in_mutini = 0
```
I'd remove the specific hemochromia subtypes from the mutini list so it doesn't get cluttered by so many of them. You can still get the unknown hemochromia.

Reply:
Done.